RocketApp/Domain/Sources/RocketsClient/Domain/URLs.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public enum URLs {
```

Comment:
Tohle sem nepatri, to bych nechal na urovni primo toho `RocketClient` adresare.

Reply:
Presunuto.